src/dbwrapper.h

```diff
@@ -44,6 +45,11 @@ const std::vector<unsigned char>& GetObfuscateKey(const CDBWrapper &w);
 
 };
 
+class CBitcoinLogger : public leveldb::Logger {
```

Comment:
CBitcoinLevelDBLogger maybe? The name is too general otherwise, we should reserve this class name for ourselves.